packages/database/src/core/RepoManager.ts

```diff
@@ -67,7 +67,7 @@ export class RepoManager {
     }
     return _staticInstance;
   }
-
+  /* eslint-disable guard-for-in */
```

Comment:
Since these functions are not on the critical part, can we use `Object.keys()` to wrap the loop variable?